dmoj/cptbox/isolate.py

```diff
@@ -185,11 +186,15 @@ def __init__(self, read_fs, write_fs=None, writable=(1, 2)):
 
     def _compile_fs_jail(self, fs):
         if fs:
-            fs_re = '|'.join(fs)
+            fs.build()
+
+            return fs
         else:
-            fs_re = '(?!)'  # Disallow accessing everything by default.
+            dummy = FilesystemPolicy()
```

Comment:
Does this work? `FilesystemPolicy` is expecting an argument but we're not providing one here.

Reply:
No, this will fail. It needs to be fixed.

Reply:
On a related note, is it preferable for FilesystemPolicy to use a variadic argument list or take a list?

Reply:
Prefer named arguments over varargs for things that aren't typed by a human. Varargs mostly just make things more syntactically pleasing, at the cost of explicitness.